### PR TITLE
This is what it looks like when I type

### PR DESCRIPTION
**Pull request by ghost**
Great places in Ann Arbor...

You really need to check it out @Rconroy

### CODE REVIEW

**Comment by ghost** [comment]
That code looks awful.

File: agoetzke_ [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Ann Arbor MI
+
+	### Great places to eat
```
Comment (githubteacher) [inline agoetzke_:3]: @agoetzke I love Italian food (hint hint 😜)

File: agoetzke_ [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+	### Fun things to do
+	
+		-Changing X Code online
+		-Not a very smart thing to do
+		-It is not very fun
+
```
Comment (Rconroy) [inline agoetzke_:10]: @agoetzke  Sounds boring.

**Comment by Rconroy** [comment]
Looks good :shipit:

File: agoetzke_ (hunk 1 above)
Comment (ghost) [reply agoetzke_:3]: Gratzi!    @githubteacher

**Pull request merged**